install-bind.bash

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+if [ $# -ne 1 ]; then
+    echo "supply target dir"
+    exit 1
+fi
+TARGET_DIR=$1
+BIND_URL="https://ftp.isc.org/isc/bind9/9.12.0rc1/bind-9.12.0rc1.tar.gz"
```

Comment:
Why do you compile it? And not just grab the package or other binary?

Reply:
To get ed25519 support.